serverless/functions/create-stream.js

```diff
@@ -362,6 +367,8 @@ module.exports.handler = async (context, event, callback) => {
       viewers_map: 'viewers',
       raised_hands_map: `raised_hands`,
     },
+    room_sid: room.sid,
```

Comment:
Just curious, why are we adding room sid here?

Reply:
@timrozum mentioned that for iOS it would be best if all participants (host, speaker, viewer) get the room sid the same way for consistency. I figured that this small change wouldn't have any impact for web. What do you think though @timmydoza?